python/gauss2d/fit/parameters.cc

```diff
@@ -23,40 +23,31 @@
 
 #include <pybind11/attr.h>
 #include <pybind11/pybind11.h>
+#include <pybind11/operators.h>
 #include <pybind11/stl.h>
 
-#include <limits>
 #include <memory>
 #include <string>
 
-#include "gauss2d/centroid.h"
-#include "gauss2d/ellipse.h"
-#include "gauss2d/fit/centroidparameters.h"
-#include "gauss2d/fit/ellipseparameters.h"
 #include "gauss2d/fit/parameters.h"
 #include "gauss2d/fit/transforms.h"
 #include "parameters/transform.h"
-#include "parameters/unit.h"
+
+#include "utils.h"
 
 namespace py = pybind11;
 using namespace pybind11::literals;
 
-// This is so annoying that I'm considering borrowing a solution for static string
-template<typename T>
-constexpr std::string_view suffix_type();
-
-template<> constexpr std::string_view suffix_type<double>() { return "D";}
-
-template<typename T>
-std::string suffix_type_str() { return std::string(suffix_type<T>()); }
+namespace g2f = gauss2d::fit;
 
 template<typename T>
 constexpr std::string_view limits_name();
 
 template<> constexpr std::string_view limits_name<double>() { return ", gauss2d.fit.LimitsD";}
 
-// Yes, this class exists solely to hold a string so that it can be initialized first
-// in an inheritance hierarchy. Solve this better if you can.
+// This class exists solely to hold a string so that it can be
+// initialized first in an inheritance hierarchy.
+// TODO: Find a better solution.
```

Comment:
Private inheritance from `std::string` might be another option, though I'm not sure it's a better one.

Reply:
Hmm. I'm not sure either, but I'll add a note to consider it.